sale_order_report_hide_tax/tests/test_show_tax_column.py

```diff
@@ -0,0 +1,125 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestShowTaxColumnInReport(TransactionCase):
```

Comment:
Use `BaseCase` by default if you don't need tracking
```suggestion
class TestShowTaxColumnInReport(BaseCase):
```